config/services.php

```diff
@@ -65,7 +65,7 @@
         'client_credentials' => [
             'client_id' => env('NORTHSTAR_CLIENT_ID'),
             'client_secret' => env('NORTHSTAR_CLIENT_SECRET'),
-            'scope' => ['user', 'admin'],
+            'scope' => ['admin', 'user', 'client', 'activity', 'write'],
```

Comment:
@sbsmith86 Before we deploy this to production, let's remove the `client` scope! That allows an application to read/write OAuth client details, which we don't want most applications (aside from Aurora) to be able to do.